general-doc/thyra/index.rst

```diff
@@ -0,0 +1,68 @@
+Thyra
+=====
+
+The project
+-----------
+
+Thyra is a secured gate to Massa blockchain and pure decentralization.
+Developped by Massa Labs, this application provides a user-friendly way of accessing,
+using and building on the blockchain while preventing users from possible attack or hack.
+Indeed, because Thyra is installed on your own device there is absolutely nothing
+and no one between you and the blockchain. Security and decentralization are the app's pillars.
+
+Concretely, Thyra enables every user to have a custom experience with the blockchain.
+Once the app installed, users will be able to activate specific functionalities in a plug-and-play mode
+that will provide them what they need. For instance, you will be able to run a distant node in a couple of clicks
+or if you prefer you will be able to create and publish a website on the blockchain.
+These functionalities are, technically speaking, plugins that can be developed by all developers that wish to.
+All details about how you can contribute will be added very soon.
+
+The installation
+----------------
+
+To install Thyra now, you just have to download the app based on your device.
+
+* Desktop
+
+  * `Windows <https://github.com/massalabs/thyra/releases/latest/download/thyra-installer_windows_amd64.exe>`_
+  * MacOS
+
+    * `Mac M1/M2 <https://github.com/massalabs/thyra/releases/latest/download/thyra-installer_darwin_arm64.tar.gz>`_
+    * `AMD <https://github.com/massalabs/thyra/releases/latest/download/thyra-installer_darwin_amd64.tar.gz>`_
+
+  * `Linux <https://github.com/massalabs/thyra/releases/latest/download/thyra-installer_linux_amd64.tar.gz>`_
+
+* Mobile
+
+  * Soon!
+
+
+The applications
+----------------
+
+* Decentralized web :ref:`Decentralized Web<general-decentralized-web>`
+
+Store your website on Massa blockchain in the right way, the way it was meant be.
+
+* Node-manager
+
+Install a Massa node on a distant server and start staking in a couple of clicks.
+
+* More to come ...
+
+
+The technical details
+---------------------
+
+Thyra's architecture will be published soon.
+
+
+
+To go further
+-------------
+
+
+* `Thyra's GitHub <https://github.com/massalabs/thyra>`_
+* `Contributing to Thyra <https://github.com/massalabs/thyra/blob/main/CONTRIBUTING.md>`_
+* `MassaLabs GitHub <https://github.com/massalabs/massa>`_
```

Comment:
It's really weird to add Github link and Massa discord here. It's already in a dedicated section of the documentation. Otherwise we would have to put it everywhere.

Reply:
Done